examples/local/etcd-up.sh

```diff
@@ -41,7 +41,8 @@ echo "add /vitess/test"
 # If the node already exists, it's fine, means we used existing data.
 echo "add test CellInfo"
 
-"${VTROOT}"/bin/vtctl ${TOPOLOGY_FLAGS} AddCellInfo \
+# shellcheck disable=SC2086
+"${VTROOT}"/bin/vtctl ${TOPOLOGY_FLAGS[@]} AddCellInfo \
```

Comment:
Based on the example given here: https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions

double quoting should work in this case because `TOPOLOGY_FLAGS` is an array.

Have you tried that?

Reply:
i try it ,  add it let add double quoting,  i add it like this
"${VTROOT}"/bin/vtctl "${TOPOLOGY_FLAGS[@]}" AddCellInfo,
but it case a problem.
![screenshot from 2018-04-16 14-45-43](https://user-images.githubusercontent.com/2627712/38793692-dc3435be-4184-11e8-8913-4504e56095c0.png)
Inexplicably a pair of single quotes.